cypress/integration/ampSpec.js

```diff
@@ -42,6 +42,17 @@ describe('AMP Tests on a .amp page', () => {
     ampScript.should('have.attr', 'src', 'https://cdn.ampproject.org/v0.js');
   });
 
+  it('should contain the AMP boilerplate', () => {
+    // .eq(1) gets the amp <script> as the first loaded is a Cypress <script>
```

Comment:
But you're getting `style`, not `script` - was this comment copied & pasted?

Reply:
Ooops, yep, looks like James copied the test just above.